charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -54,32 +54,8 @@ spec:
             value: "kubernetes.default.svc"
           - name: KUBERNETES_SERVICE_PORT
             value: "443"
-          - name: HELM_SCALAR_DB_CONTACT_POINTS
-            value: "{{ .Values.scalardbCluster.storageConfiguration.contactPoints }}"
-          - name: HELM_SCALAR_DB_CONTACT_PORT
-            value: "{{ .Values.scalardbCluster.storageConfiguration.contactPort }}"
```

Comment:
These environment variables are related to the deprecated `scalardbCluster.storageConfiguration.*` configurations.